### PR TITLE
Move Axelar Icons

### PR DESCRIPTION
**Pull request by blurrytweaks**
Hey Fig these are the notes taken while doing the job!

```
ape - have it with the new tokens *added regardless
archway - arch.svg
btcb - btc.svg
dev - is a glmr icon img *added regardless
dot-planck - *added an inverted dot img.
fetch - fet.svg
injective - inj.svg
lqdr- have it with the new tokens *added regardless
stuatom - statom.svg *same img
susdz - have it with the new tokens *added regardless.
tsushi - axl icon - to be updated.
ust - ustc.svg
added b&w versions for wbnb, wdai, wmai and wmnt.
wdev - wglmr.svg
wglmr - wglmr-w.svg
```

### CODE REVIEW

File: images/master/tokens/ern.svg
Comment (ecdsafu) [inline images/master/tokens/ern.svg:1]: This one I think we can revert. It will look better with a white outline in our drip-downs

Comment (blurrytweaks) [reply images/master/tokens/ern.svg:1]: haha gotcha! now I tend to keep the git bash cmd open for these quick cases, for any other edits feel free to review

Comment (ecdsafu) [reply images/master/tokens/ern.svg:1]: Nice nice, looks good!

**Review by ecdsafu — APPROVED** [review]
lgtm

**Pull request merged**